src/NetAnalyzers/Directory.Build.props

```diff
@@ -6,5 +6,8 @@
     <FxCopAnalyzersProject Condition="$(MSBuildProjectName.StartsWith('Microsoft.CodeQuality')) OR $(MSBuildProjectName.StartsWith('Microsoft.NetCore')) OR $(MSBuildProjectName.StartsWith('Microsoft.NetFramework'))">true</FxCopAnalyzersProject>
 	<DefineConstants Condition="'$(FxCopAnalyzersProject)' == 'true'">$(DefineConstants),FXCOP_ANALYZERS</DefineConstants>
     <DefineConstants Condition="'$(FxCopAnalyzersProject)' != 'true'">$(DefineConstants),NET_ANALYZERS</DefineConstants>
+
+    <!-- Override the directory for BannedSymbols.txt -->
+    <BannedSymbolsDir>$(MSBuildThisFileDirectory)</BannedSymbolsDir>
```

Comment:
❓ Can we not use both files?

Reply:
Good catch, I didn't know the analyzer accepted multiple banned files! Fixed now.